script.js

```diff
@@ -119,13 +131,40 @@ function clearGameBoard(event) {
   disableGameSpaces(false);
 }
 
+function insertPlayerXName(userInput) {
+  let p1Name = document.getElementById("P1Name");
+  let inputName = document.getElementById("P1NameInput").value;
+
+  p1Name.innerHTML = inputName;
+  document.getElementById("P1NameInput").style.visibility = "hidden";
```

Comment:
you already have this element saved in a variable called `inputName` from above so you can replace all the
`document.getElementByID("P1NameInput)` with `inputName`